ui/app.js

```diff
@@ -3921,7 +3924,12 @@ $(document).ready(() => {
         $(".incidents-nav-item").hide();
         $(".dmv-nav-item").hide();
         $(".cams-nav-item").hide();
-        $("#reports-officers-involved-tag-title").html("EMS Involved");
+        $("#reports-officers-involved-tag-title").html(

```

Comment:
I really dislike that it uses the same handling as reports for officers and bolos...